std-bits/base/src/main/java/org/enso/base/cache/LRUCache.java

```diff
@@ -28,22 +28,46 @@
  * deleting entries to make space for new ones. All cache files are set to be deleted automatically
  * on JVM exit.
  *
+ * <p>Limits should be set with environment variables: ENSO_LIB_HTTP_CACHE_MAX_FILE_SIZE_MEGS --
+ * single file size, in megs ENSO_LIB_HTTP_CACHE_MAX_TOTAL_CACHE_LIMIT -- total cache size, in megs
```

Comment:
Can we use full names instead of abbreviations in the docs? Or at least let's use the standard `MB` abbreviation. `Megs` sounds very casual and may not be understandable to everyone IMHO.

Reply:
Done